bootloaders/riotboot_dfu/main.c

```diff
@@ -74,9 +73,9 @@ void kernel_init(void)
         }
     }
 
-    /* Init ztimer before starting DFU mode */
-    ztimer_init();
-
+    if (IS_USED(MODULE_ZTIMER)) {
+        ztimer_init();
+    }
```

Comment:
If ztimer is not used anymore, we can just drop this

Reply:
Unfortunately no.
Both periph_usbdev for STM32 need it. So we still need it. But this way, nRF52 and SAM0 won't use it and will save some ROM.